actors/builtin/market/policy.go

```diff
@@ -9,6 +9,9 @@ import (
 // DealUpdatesInterval is the number of blocks between payouts for deals
 const DealUpdatesInterval = 100
 
+var ProvCollateralPercentSupplyNum = big.NewInt(5)
```

Comment:
These should have a comment explaining the formula they are used in and meaning (if that isn't too hard to describe).